### DISCUSSION TITLE
Webhooks not firing under version 3.5 onwards

### DISCUSSION THREAD

**Discussion by amineff** [saleor/saleor#10263] [General] [answered]
I have spent hours on this I have tried to activate every option but saleor is not emitting any webhooks, the feature however is working well with version 2.11.x

UPDATE : I have this issue on the heroku instance but not the local instance running on docker.
![screenshot](https://user-images.githubusercontent.com/71326309/181795759-e8c61752-53ec-4ce4-ab43-b3d166108a00.png)

**Comment by korycins** [comment, marked as answer]
Hey,
- make sure that your app has expected permissions assigned. Like, it will not call order related webhooks without `MANAGE_ORDERS`.
- make sure that your app is active
- make sure that you have celery worker that can process outcoming webhooks.

**Reply by patrys** [reply]
My guess is that there is no Celery worker running so asynchronous tasks get queued but never get picked up. Consult Heroku's docs to see how to run non-web workers and have it run `celery -A saleor --app=saleor.celeryconf:app worker --beat`.

**Reply by amineff** [reply]
Yeah thanks, it was the  Celery worker which were not configured
